drv/local-vpd/src/lib.rs

```diff
@@ -121,19 +125,31 @@ pub fn read_config_into(
     i2c_task: TaskId,
     tag: [u8; 4],
     out: &mut [u8],
+) -> Result<usize, LocalVpdError> {
+    match read_config_inner(i2c_task, tag, out) {
+        Ok(n) => Ok(n),
+        Err(e) => {
```

Comment:
This'd be a use-case for [`inspect_err`](https://doc.rust-lang.org/std/result/enum.Result.html#method.inspect_err) if it were stable.